package.json

```diff
@@ -101,34 +101,12 @@
     "@sf/login": "npm:@salesforce/plugin-login@1.0.6",
     "tslib": "^2.3.1"
   },
-  "pinnedDependencies": [
-    "@sf/config@sf",
-    "@sf/deploy-retrieve",
-    "@sf/drm",
-    "@sf/env",
-    "@sf/functions",
-    "@sf/gen",
-    "@sf/info",
-    "@sf/login",
-    "@salesforce/plugin-telemetry",
-    "@oclif/plugin-autocomplete",
-    "@oclif/plugin-help",
-    "@oclif/plugin-not-found",
-    "@oclif/plugin-plugins",
-    "@oclif/plugin-update",
-    "@oclif/plugin-version"
-  ],
-  "resolutions": {
```

Comment:
@mdonnalley shouldn't these remain?

Reply:
@peternhale maybe - I just wanted to see if the build passed without them